### PR TITLE
[BUG] The output_index field is not supported in 8.x

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3177.

Preview:
- [Alert notification placeholders](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/rules-ui-create.html#rule-action-variables): Removed the `{{context.rule.output_index}}` placeholder param from the list of replaceable params that can be used.
- [Create rule](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/rules-api-create.html): 
    - Removed the `output_index` field from the list of optional fields for all rule types.
    - Removed the `{{context.rule.output_index}}` placeholder param from the list of replaceable params that can be used.
    - Removed the `output_index` field from various response payload examples.
- [Update rule](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/rules-api-update.html): 
    - Removed the `output_index` field from the list of optional fields for all rule types.
    - Removed the `output_index` field from the response payload example.
- [Bulk update rules](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/bulk-actions-rules-api.html#_response_payload_8): Removed the `output_index` from the response payload examples.
-  Removed the `output_index` field from the response payload examples in the following topics:
    - [Find rules](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/rules-api-find.html#_response_payload_3)
    - [Get rule](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/rules-api-get.html#_response_payload_2)
- [8.0 release notes](https://security-docs_3178.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.0.0.html#deprecations-8.0.0): Added a section for deprecation features and mentioned that the field is no longer supported for rule APIs and is also not included in the rules export file.

### CODE REVIEW

File: docs/release-notes/8.0.asciidoc [hunk 1]
```diff
@@ -113,6 +113,11 @@ A new Lucene 9 validation change may cause event correlation (EQL) rule errors w
 . Refresh the page to verify the deprecation was resolved, then return to the guided steps on the Upgrade Assistant page.
 // end::breaking-changes[]
 
+[discrete]
+[[deprecations-8.0.0]]
+==== Deprecations
+* The `output_index` parameter is no longer supported for the APIs that create and update rules. In addition, the `output_index` field is not included in the exported rule file.
```
Comment (WafaaNasr) [inline docs/release-notes/8.0.asciidoc:119]: `the `output_index` field is not included in the exported rule file.` 
Unfortunately, this part is not yet fixed, I created this bug ticket https://github.com/elastic/kibana/issues/155355 to be fixed by the team. Until then the `output_index` will still be part of the exported rule but it doesn't have any impact

Comment (nastasha-solomon) [reply docs/release-notes/8.0.asciidoc:119]: Gotcha, thank you! I'll remove it from the release notes summary then. Would you recommend any other changes to the first sentence or does it accurately describe the deprecated field?

```suggestion
* The `output_index` parameter is no longer supported for the APIs that create and update rules. 
```

**Comment by nastasha-solomon** [comment]
> I found this [article](https://www.elastic.co/guide/en/security/8.7/signals-index-template.html) still mentions the `output_index` in defining the template of the Alert, do you think we need to update it ?

Yes, we should definitely remove the field from the alert template. Good catch, thank you!

File: docs/release-notes/8.0.asciidoc (hunk 1 above)
Comment (WafaaNasr) [reply docs/release-notes/8.0.asciidoc:119]: Thank you Nastasha! All is good 👍

**Review by WafaaNasr — APPROVED** [review]
LGTM!! Thanks Nastasha ^_^

**Review by benironside — APPROVED** [review]
LGTM!

**Pull request merged**